gravitee-am-common/src/main/java/io/gravitee/am/common/utils/ConstantKeys.java

```diff
@@ -75,6 +75,7 @@ public interface ConstantKeys {
     String _CSRF = "_csrf";
     String SOCIAL_PROVIDER_CONTEXT_KEY = "socialProviders";
     String INTERNAL_PROVIDER_CONTEXT_KEY = "identityProviders";
+    String URL_HASH_PARAMETER = "urlhash";
```

Comment:
```suggestion
    String URL_HASH_PARAMETER = "urlHash";
```
The constant you moved here had a capital H

Reply:
There is probably something wrong with the JEST test as this error prevent me to authenticate. After the authentication on the OIDC IDP domain, the callback fails because the urlHash param can't be extracted (due to the syntax error).

Reply:
That's inconsistent with the default template that submits this parameter (`login_callback.html`)... Has the 'token' / 'id token' flow ever actually worked?

Reply:
Yes it works because the form post the parameters using the fields name which is initialiezd with the capital H.